ci/scripts/config.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+user_id=`id -u`
+
+# we want to snapshot the environment of the config user
+# if [ $user_id -eq 0 -a -z "$RUNNER_ALLOW_RUNASROOT" ]; then
+#     echo "Must not run with sudo"
+#     exit 1
+# fi
```

Comment:
Commented this part to enable running as root.
I know this is for security reasons, but since the runner runs in docker container, I think it's already isolated to be save enough (at least for now)
We'll need to a lot other work to create new user and make sure it runs safely, which I'm not bothered to do.